ratbagd/ratbagd-device.c

```diff
@@ -137,6 +137,30 @@ static int ratbagd_device_get_device_name(sd_bus *bus,
 	return 0;
 }
 
+static int ratbagd_device_get_device_type(sd_bus *bus,
+					  const char *path,
```

Comment:
Wrong indent. Use the .editorconfig file (there is a vscode extension :]).

Reply:
are you sure? its the same on all other function definitions. also it looks fine in vscode, its only off on github...

![Screenshot_20200301_105418](https://user-images.githubusercontent.com/16562333/75623525-09ca2780-5bab-11ea-8c05-26f12a4ce587.png)

Reply:
Sorry, you're right. I thought github would use editorconfig, so nvm.

Reply:
it's an issue with rendering diffs, not with whether you use editorconfig. Specifically, the `+ ` at the beginning likes to upset things so it's quite common that diffs aren't correctly aligned even though the code is.